### PR TITLE
No Man's Lava Day One Patch

### PR DESCRIPTION
**Pull request by Iamgoofball**
Revolutionizes the gaming universe with the nearly infinite possibilities of procedural generation. Explore the strange new planets with your friends, but the possibility of finding them is low.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
Hey there! Just wanted to let you know that your code failed the travis build! This usually means that your code doesn't compile, try compiling it again in dream maker to see what's giving you the problem! Good luck!

File: code/controllers/master.dm [hunk 1]
```diff
@@ -60,6 +63,15 @@ var/CURRENT_TICKLIMIT = TICK_LIMIT_RUNNING
 			init_subtypes(/datum/subsystem, subsystems)
 		Master = src
 
+	procedural_generation_mobs = color2hex(randomColor(1))
+	world << "<span class='boldannounce'>[procedural_generation_mobs]</span>"
+	procedural_generation_floors = color2hex(randomColor(1))
+	world << "<span class='boldannounce'>[procedural_generation_floors]</span>"
+	procedural_generation_walls = color2hex(randomColor(1))
+	world << "<span class='boldannounce'>[procedural_generation_walls]</span>"
+
```
Comment (MrStonedOne) [inline code/controllers/master.dm]: Pop this in a subsystem

Comment (Iamgoofball) [inline code/controllers/master.dm]: i believe this was an artistic choice by cheridan

**Comment by GunHog** [comment]
What does this even do? Randomly change mob, wall, and floor colors?  Please run this code and take a few pictures of the results, I am curious to see how it affects things.

**Comment by lordpidey** [comment]
@TechnoAlchemisto , the code does compile, it just has a warning for an unused variable.

**Comment by TechnoAlchemisto** [comment]
it was more or less a joke

**Comment by AnturK** [comment]
Good one.

**Pull request closed without merging**